pandas/tests/series/test_replace.py

```diff
@@ -243,6 +243,13 @@ def test_replace_string_with_number(self):
         expected = pd.Series([1, 2, 3])
         tm.assert_series_equal(expected, result)
 
+    def test_repace_intertwined_key_value_dict(self):
+        # GH 20656
```

Comment:
can you add a 1-liner explaining the test in a bit more detail

Reply:
typo repace --> replace (?)